src/jsMain/kotlin/baaahs/jsImpl.kt

```diff
@@ -42,7 +42,7 @@ private fun logMessage(level: String, message: String, exception: Throwable?) {
         "ERROR" -> console.error(message, exception)
         "WARN" -> console.warn(message, exception)
         "INFO" -> console.info(message, exception)
-        "DEBUG" -> console.log(message, exception)
+        "DEBUG" -> console.asDynamic().debug(message, exception)
```

Comment:
I don't understand why the kotlin stdlib doesn't support `console.debug`...

https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/js/src/kotlin/debug.kt

It's totally a thing! https://developer.mozilla.org/en-US/docs/Web/API/Console/debug

Reply:
Yep, agreed. :-p